chocolatey/choco-theme
# (#285)(#363) Add 12 Days of Chocolatey Packaging Banner

## Description Of Changes
* This adds a top banner to promote the 12 Days of Chocolatey Packaging.
* Pins Bootstrap types version

## Motivation and Context
We want to promote the 12 Days of Chocolatey Packaging over the holidays.

## Testing
1. Run the website from this PR: https://github.com/chocolatey/chocolatey.org/pull/295
2. Ensure the top banner appears with the correct text, is green, and takes you to the blog when clicked.

### Operating Systems Testing
n/a

## Change Types Made

## Change Checklist


## Related Issue
* https://github.com/chocolatey/choco-theme/issues/285
* https://github.com/chocolatey/choco-theme/issues/363
* https://github.com/chocolatey/chocolatey.org/issues/294
* https://github.com/chocolatey/docs/issues/923
* https://github.com/chocolatey/home/issues/288
* https://gitlab.com/chocolatey/community-infrastructure/community.chocolatey.org/-/issues/1241
